### ISSUE TITLE
Block id is not forwarded to envOptions

### ISSUE THREAD

**Issue by chemezov** [luyadev/luya-module-cms#376]

**Comment by nadar** [comment]
https://github.com/luyadev/luya-module-cms/blob/master/src/models/NavItemPage.php

**Comment by chemezov** [comment]
@nadar yes, i send a PR https://github.com/luyadev/luya-headless-cms-api/pull/7, but i think problem was in luyadev/luya-headless-cms-api repo only, not in luya-module-cms, but thanks for the tip! Code from https://github.com/luyadev/luya-module-cms/blob/master/src/models/NavItemPage.php helped a lot!
